test/selenium/lib/setup_webapp.js

```diff
@@ -2,19 +2,68 @@
 
 var SignedInPageView = require('./view/signed_in/view.js');
 var SetUpView = require('./view/sign_up/view.js');
+var MainView = require('./view/app_main/view.js');
+const fs = require('fs');
+const path = require('path');
+const spawn = require('child_process').spawn;
+const Config = require('config-js');
+
+var config = new Config('./test/integration/lib/config/foxbox.js');
+var foxboxInstance;
+var FOXBOX_STARTUP_WAIT_TIME_IN_MS = 5000;
 
 function SetUpWebapp(driver) {
     this.driver = driver;
 }
 
 SetUpWebapp.prototype = {
     getSignInPage : function() {
-        return new SignedInPageView(this.driver);
+      return new SignedInPageView(this.driver);
     },
 
     getSetUpView : function() {
-        return new SetUpView(this.driver);
-    }
+      return new SetUpView(this.driver);
+    },
+
+    getAppMainView : function() {
+      console.log('getAppMainView');
+      return new MainView(this.driver);
+    },
+
+    _removeFileIfItExists: function(filename,errMsg) {
+      try {
+          fs.unlinkSync(path.join(process.env.HOME, 
+          filename));
+      } catch (e) {
+          if (e.code === 'ENOENT') {
+            console.log(errMsg);
+          }
+            }
+    },
+
+    removeUsersDB: function () {
+      this._removeFileIfItExists('/.local/share/foxbox/users_db.sqlite',
+      'User DB not found!');
+    }, 
+
+   fullOptionStart: function(callback) {
+        console.log('start new foxbox');
+        foxboxInstance = spawn('./target/debug/foxbox',
+        ['-c',
+        '--disable-tls']
+        );
+      setTimeout(callback, FOXBOX_STARTUP_WAIT_TIME_IN_MS);
+      return;
+    },
+
+    killFoxBox: function () {
```

Comment:
Hmm, here, you are also spawning foxbox, the spawning instance should be called separately, it should be in this method